src/app/(auth)/login/page.tsx

```diff
@@ -29,14 +30,15 @@ export default function Login() {
     if (auth.userId && !finishLogin) push(CONFIG.settings);
   }, [auth, profile, push]);
 
-  const handleSignIn = async () => {
+  const handleSignIn = async (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
     if (!auth) {
       setErrorMessage('');
       return;
     }
 
-    setEmailError(validEmail(email) ? '' : 'Invalid Email');
-    setPasswordError(password !== '' ? '' : 'Invalid Password');
+    setEmailError(validEmail(email) ? '' : 'Invalid email.');
+    setPasswordError(password !== '' ? '' : 'Invalid password.');
```

Comment:
Could we update the error messages to match the design:
<img width="588" alt="image" src="https://github.com/calblueprint/immigration-justice-project/assets/84427964/4ed10774-90d8-4fb8-92de-8e5b4d3c31fa">

Reply:
Quick note: forgot password flow has a similar error message for the email ("Could not find email"). Just messaged @kyrenetam to see if we wanna decide on one of the messages for consistency, will update the comments once there's a decision.

Reply:
sg! i'm gonna hold off on any changes for now then

Reply:
from Kyrene: go with "Email not found"